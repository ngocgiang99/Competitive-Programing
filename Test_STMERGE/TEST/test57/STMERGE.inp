4
10 8
565863140 865935074 282468056 83761321 915163779 750903292 555499361 107165514 
380108914 169599430 160249532 313505254 199051719 940444645 171756575 651156545 
580660694 140812862 795450112 267617377 469904437 503400181 809373458 597492873 
56677560 854967407 877792331 394567487 68611956 135541587 492087520 978119497 
815396542 243020420 57538373 275113387 846719789 353574696 586417713 133712298 
198070675 17159843 974947479 97320139 246616327 785825435 748083630 57102547 
751338205 697767669 521699561 105144775 922536619 636182485 724676094 916561388 
352902725 407128481 972359394 187004360 835206080 79970045 613031882 507940383 
780238194 885515983 884296491 986008566 504907247 828469820 283337146 952469138 
373393421 760019113 963236460 671466360 867444448 733737914 870370933 519586413 
5 10
899329868 383396727 821419578 170288030 942049956 133607641 699818539 198905083 404798067 220646682 
743885221 571311121 351749870 487171838 909976318 584887236 154692746 45145994 51780299 75360058 
961020924 339359235 184365023 532739543 973017657 758529518 595744593 459348290 927881845 228023056 
21099638 2181469 744926759 173263727 721826018 440407632 990808834 504387919 267341418 256586366 
946348285 295168821 426978084 976329357 514277963 691506150 13316964 16365921 215531362 27660391 
5 6
351862007 294349309 909306123 593615919 895078911 891483181 
358681692 73566002 736871343 385952957 645113096 142497551 
128109948 796745136 828269219 686820843 919344410 667593037 
996300034 785789482 266144896 898431000 629634619 426932850 
117921761 57584854 31023266 238085544 764477363 80903821 
1 10
802599083 392508544 636430736 470461872 896667070 365954102 41456128 879468428 686635553 874291058 
