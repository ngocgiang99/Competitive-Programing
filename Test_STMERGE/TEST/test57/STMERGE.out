107165514
48760029
117921761
407410230
